Source/WebCore/page/scrolling/ScrollingCoordinatorTypes.h

```diff
@@ -89,6 +89,7 @@ struct ScrollableAreaParameters {
     NativeScrollbarVisibility verticalNativeScrollbarVisibility { NativeScrollbarVisibility::Visible };
 
     ScrollbarWidth scrollbarWidthStyle { ScrollbarWidth::Auto };
+    std::optional<ScrollbarColor> scrollbarColorStyle { };
```

Comment:
```suggestion
    std::optional<ScrollbarColor> scrollbarColorStyle;
```

`std::optional` is default initialized to nullopt.

Reply:
Fixed.